docs/cookbook.rst

```diff
@@ -1,4 +1,85 @@
 Cookbook
 ========
 
-Recipes for specific use cases
+Task Recipes
+------------
+
+Run a Shell Command
+******************************
+
+.. code-block:: yaml
+
+    run_custom_command:
+        description: Greets the user
+        class_path: cumulusci.cli
+        options: 
+            command: "echo 'Hello there!"
+
+        
+
+Run a ``sfdx`` Command
+****************************
+The ``dx`` task lets you run an arbitrary ``sfdx`` command.
+You can perform this with ``cci`` on a terminal::
+
+    $ cci task run dx -o command 'force:api:limits:display'
+
+Or you can utilize the same ``class_path`` as the ``dx`` task and make a custom task that can be executed by itself or as a step in a flow.
+
+.. code-block:: yaml
+
+    dx_limits:
+        description: Display
+        class_path: cumulusci.tasks.sfdx.SFDXBaseTask
+        group: dx 
+        options: 
+            command: sfdx force:limits:api:display
+
+In this case, we actually utilize ``SFDXBaseTask``, if you would like to run a ``sfdx`` command that references an org, utilize ``SFDXOrgTask`` instead.
+
+
+
+Custom Deploy
+************************
+It is often useful to be able to define multiple custom deployment tasks that deployg a specific subset of your projects metadata.
+This is particularly true when working with `unpackaged Metadata <TODO>`_.
+
+Here is a custom task that is defined to only deploy only the metadata contained in ``unmanaged/config/reports``.
+
+.. code-block:: yaml
+
+    deploy_reports:
+        description: Deploy Reports 
+        class_path: cumulusci.tasks.salesforce.Deploy
+        options:
+            path: unmanaged/config/reports    
+
+Being able to give this task a new name makes it much more intuitive as to what the task is actually doing.
```

Comment:
```suggestion
Giving this task a descriptive name makes the task's purpose much more intuitive.
```
In sales, they say "Always be selling." In doc, we say "Always be trimming." This sentence is a prime candidate for tightening up for clarity and readability.